### PR TITLE
Add Miguel Piedrafita to the students section

### PR DESCRIPTION
**Pull request by m1guelpf**

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -213,3 +213,11 @@
   image: dwij.png
   facebook: jha.dwij
   bio: 14,Indian,Sudent Programmer
+
```
Comment (jigyasa-grover) [inline _data/students.yml]:
```
Status: Downloaded newer image for coala/base:latest
Executing section Default...
|    | [NORMAL] SpaceConsistencyBear:
|    | Line contains following spacing inconsistencies:
|    | - Trailing whitespaces.
|----|    | /app/_data/students.yml
|    |++++| /app/_data/students.yml
| 213| 213|   image: dwij.png
| 214| 214|   facebook: jha.dwij
| 215| 215|   bio: 14,Indian,Sudent Programmer
| 216|    |-  
|    | 216|+
| 217| 217| - name: Miguel Piedrafita
| 218| 218|   github: m1guelpf
| 219| 219|   facebook: m1guelpiedrafita
[INFO][17:47:06] Applied 'ShowPatchAction' on '_data/students.yml' from 'SpaceConsistencyBear'.
Executing section html...
Executing section yaml...
Executing section css...
Executing section js...
```

Trailing whitespace at Line 216 leading to build failure.

**Comment by m1guelpf** [comment]
@jig08 More work?
What do I have to do?

**Comment by jigyasa-grover** [comment]
Remove trailing whitespace at `line 216`

**Comment by m1guelpf** [comment]
@jig08 Done

**Review by jigyasa-grover — COMMENTED** [review]
* Squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students/mentors section"
* Also provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.

**Comment by m1guelpf** [comment]
@jig08 Squashed.
My changes are live at http://projects.miguelpiedrafita.com/gci16.fossasia.org/

**Pull request merged**